src/forms-opportunities/handler.js

```diff
@@ -102,6 +103,7 @@ export async function processOpportunityStep(context) {
   const latestAudit = await site.getLatestAuditByAuditType('forms-opportunities');
   await createLowConversionOpportunities(finalUrl, latestAudit, scrapedData, context);
   await createLowNavigationOpportunities(finalUrl, latestAudit, scrapedData, context);
+  await createLowFormViewsOpportunities(finalUrl, latestAudit, scrapedData, context);
```

Comment:
can you pls also make sure that the form url once detected in the initial stages of the conversion funnel dont repeat in subsequent stages.